doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -3155,7 +3154,7 @@ An :token:`r_item` can be:
 
 .. flag:: SsrOldRewriteGoalsOrder
 
-   Controls the order in which generated subgoals (side conditions)
+   This flag controls the order in which generated subgoals (side conditions)
```

Comment:
```suggestion
   Controls the order in which generated subgoals (side conditions)
```

Our style is to not include boilerplate phrases such as "This flag" when describing the flag.